### ISSUE TITLE
Switching query windows doesn't switch results windows

### ISSUE THREAD

**Issue by Charles-Gagnon** [microsoft/vscode-mssql#1587]
Switching between query windows isn't changing the results window which makes it hard to follow which results actually correspond to the window you're looking at.

**Comment by Kenobi-the-2nd** [comment]
Agreed. This is a much needed feature. Clicking on a query tab should open the corresponding results tab. This would make navigating multiple queries/results much easier.

**Comment by Charles-Gagnon** [comment]
Oh, didn't check whether this was already opened. Closing this as a duplicate of #1480
